tests/helpers.js

```diff
@@ -47,6 +48,8 @@ class FuzzTest {
 		verbose,
 		coverage,
 		expectedErrors,
+		asJson,
```

Comment:
Btw., why are our tests not written in TypeScript?

Reply:
Jazzer.js did not support TS in the beginning, hence integration tests and the like were written in JS. We can and probably should convert all tests possible to TS.